src/index.html

```diff
@@ -3,9 +3,46 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Capstone setup</title>
+    <link href="//fonts.googleapis.com/css?family=Allison&subset=latin" rel="stylesheet" type="text/css">
+    <script src="https://kit.fontawesome.com/870e4168d3.js" crossorigin="anonymous"></script>
+    <title>Capstone</title>
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+    <header class="header">
+        <div class="logo"><img class="logo" src="https://www.kindpng.com/picc/m/193-1932544_indian-food-beverage-logo-hd-png-download.png" alt="restaurant-logo"/></div>
+        <div class="links">
+            <div class="link"><a href="#">KC99 Restaurant Food list</a></div>
+        </div>
+    </header>
+    <main>
+        <div id="root" class="cards container">
+        </div>         
+    </main>
```

Comment:
- [ ] Please kindly make sure there is a counter in the homepage that displays the count of the items displayed